# Analysis of Apps

- This project is to analyze the apps and their revenue potential
- The goal is to help developers to understand what kinds of apps are likely to attract more users on Google Play and the App Store.

In [159]:
def explore_data(dataset, start, end, rows_and_columns=False):
    dataset_slice = dataset[start:end]    
    for row in dataset_slice:
        print(row)
        print('\n') # adds a new (empty) line after each row

    if rows_and_columns:
        print('Number of rows:', len(dataset))
        print('Number of columns:', len(dataset[0]))

In [160]:
from csv import reader

### The Google Play data set ###
opened_file = open('googleplaystore.csv')
read_file = reader(opened_file)
android = list(read_file)
android_header = android[0]
android = android[1:]

### The App Store data set ###
opened_file = open('AppleStore.csv')
read_file = reader(opened_file)
ios = list(read_file)
ios_header = ios[0]
ios = ios[1:]

In [161]:
android_header

['App',
 'Category',
 'Rating',
 'Reviews',
 'Size',
 'Installs',
 'Type',
 'Price',
 'Content Rating',
 'Genres',
 'Last Updated',
 'Current Ver',
 'Android Ver']

In [162]:
explore_data(android, 0, 11, rows_and_columns=True)

['Photo Editor & Candy Camera & Grid & ScrapBook', 'ART_AND_DESIGN', '4.1', '159', '19M', '10,000+', 'Free', '0', 'Everyone', 'Art & Design', 'January 7, 2018', '1.0.0', '4.0.3 and up']


['Coloring book moana', 'ART_AND_DESIGN', '3.9', '967', '14M', '500,000+', 'Free', '0', 'Everyone', 'Art & Design;Pretend Play', 'January 15, 2018', '2.0.0', '4.0.3 and up']


['U Launcher Lite – FREE Live Cool Themes, Hide Apps', 'ART_AND_DESIGN', '4.7', '87510', '8.7M', '5,000,000+', 'Free', '0', 'Everyone', 'Art & Design', 'August 1, 2018', '1.2.4', '4.0.3 and up']


['Sketch - Draw & Paint', 'ART_AND_DESIGN', '4.5', '215644', '25M', '50,000,000+', 'Free', '0', 'Teen', 'Art & Design', 'June 8, 2018', 'Varies with device', '4.2 and up']


['Pixel Draw - Number Art Coloring Book', 'ART_AND_DESIGN', '4.3', '967', '2.8M', '100,000+', 'Free', '0', 'Everyone', 'Art & Design;Creativity', 'June 20, 2018', '1.1', '4.4 and up']


['Paper flowers instructions', 'ART_AND_DESIGN', '4.4', '167', '5.6M', '50,000+

In [163]:
ios_header

['id',
 'track_name',
 'size_bytes',
 'currency',
 'price',
 'rating_count_tot',
 'rating_count_ver',
 'user_rating',
 'user_rating_ver',
 'ver',
 'cont_rating',
 'prime_genre',
 'sup_devices.num',
 'ipadSc_urls.num',
 'lang.num',
 'vpp_lic']

In [164]:
explore_data(ios, 0, 15, rows_and_columns=True)

['284882215', 'Facebook', '389879808', 'USD', '0.0', '2974676', '212', '3.5', '3.5', '95.0', '4+', 'Social Networking', '37', '1', '29', '1']


['389801252', 'Instagram', '113954816', 'USD', '0.0', '2161558', '1289', '4.5', '4.0', '10.23', '12+', 'Photo & Video', '37', '0', '29', '1']


['529479190', 'Clash of Clans', '116476928', 'USD', '0.0', '2130805', '579', '4.5', '4.5', '9.24.12', '9+', 'Games', '38', '5', '18', '1']


['420009108', 'Temple Run', '65921024', 'USD', '0.0', '1724546', '3842', '4.5', '4.0', '1.6.2', '9+', 'Games', '40', '5', '1', '1']


['284035177', 'Pandora - Music & Radio', '130242560', 'USD', '0.0', '1126879', '3594', '4.0', '4.5', '8.4.1', '12+', 'Music', '37', '4', '1', '1']


['429047995', 'Pinterest', '74778624', 'USD', '0.0', '1061624', '1814', '4.5', '4.0', '6.26', '12+', 'Social Networking', '37', '5', '27', '1']


['282935706', 'Bible', '92774400', 'USD', '0.0', '985920', '5320', '4.5', '5.0', '7.5.1', '4+', 'Reference', '37', '5', '45', '1']


['5538347

As we only build apps that are free to download and install, and apps that are directed toward an English-speaking audience, we will need to remove:
- Non-English apps
- Non-free apps

In [165]:
#Finding the row in Google Play Data
android[10472]

['Life Made WI-Fi Touchscreen Photo Frame',
 '1.9',
 '19',
 '3.0M',
 '1,000+',
 'Free',
 '0',
 'Everyone',
 '',
 'February 11, 2018',
 '1.0.19',
 '4.0 and up']

In [166]:
#del the row with missing data
del android[10472]

There are no issues with App Store data set.

In [167]:
#finding instagram duplicates
for app in android:
    name = app[0]
    if name == "Instagram":
        print(app)

['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577446', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66577313', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']
['Instagram', 'SOCIAL', '4.5', '66509917', 'Varies with device', '1,000,000,000+', 'Free', '0', 'Teen', 'Social', 'July 31, 2018', 'Varies with device', 'Varies with device']


There are duplicates in Google Play data set. Below is the number of duplicate apps in the data set and a few examples. One reason could be collection of ratings data.

In [168]:
duplicate_apps = []
unique_apps = []

for app in android:
    name = app[0]
    if name in unique_apps:
        duplicate_apps.append(name)
    else:
        unique_apps.append(name)

print('Number of duplicate apps:', len(duplicate_apps))
print('\n')
print('Examples of duplicate apps:', duplicate_apps[:15])

Number of duplicate apps: 1181


Examples of duplicate apps: ['Quick PDF Scanner + OCR FREE', 'Box', 'Google My Business', 'ZOOM Cloud Meetings', 'join.me - Simple Meetings', 'Box', 'Zenefits', 'Google Ads', 'Google My Business', 'Slack', 'FreshBooks Classic', 'Insightly CRM', 'QuickBooks Accounting: Invoicing & Expenses', 'HipChat - Chat Built for Teams', 'Xero Accounting Software']


We won't be removing the duplicates randomly. As we looked closely at the data, it is shown that the duplicates can be caused by ratings so we will be taking apps with highest ratings counts (most recent).

**Explanation for below cell:**

- We have created an empty dictionary to store the app and the max no of reviews it has
- We loop through the android data to find if the name is in dictionary (it won't if it is first time), then we checked the number of reviews it has against the data in dictionary (imagine if an app has two duplicates, one has 2 reviews and another one has 3 reviews). 
- If the number of reviews in dictionary is lesser than the one in data, we will update the count in dictionary to reflect the max ratings. Using above scenario, the app review count will become 3.
- Another simple stage is to find if the app name is already in the dictionary, or else add the name and number of reviews it has.

In [169]:
reviews_max = {}

for row in android:
    name = row[0]
    n_reviews = float(row[3])
    if name in reviews_max and reviews_max[name] < n_reviews:
        reviews_max[name] = n_reviews
    elif name not in reviews_max:
        reviews_max[name] = n_reviews

print(len(reviews_max))

9659


**Explanation for below cell:**

- We created two empty lists, android_clean and already_added
- Remember we already had a dictionary of apps and its max reviews in above. We loop through the android data set, match the data against dictionary. 
- If the row data in android data set has name in dictionary and its number of reviews is the same as the one in dictionary, and its name is not already added before (this is important as we don't want another duplicate scenario), we will add this app information (whole row of data) into android_clean list. We will also add the app name in already_added list to make sure we have the count of this.

In [170]:
android_clean = []
already_added = []

for row in android:
    name = row[0]
    n_reviews = float(row[3])
    if (name in reviews_max) and (n_reviews == reviews_max[name]) and (name not in already_added):
        android_clean.append(row)
        already_added.append(name)

print(len(android_clean))
print(len(already_added))

9659
9659


# Checking Non-English Apps

In [171]:
def english_char_check(a_string):
    for character in a_string:
        if ord(character) > 127:
            return False
    
    return True

In [172]:
english_char_check('Instagram')

True

In [173]:
english_char_check('爱奇艺PPS -《欢乐颂2》电视剧热播')

False

In [174]:
english_char_check('Docs To Go™ Free Office Suite')

False

In [175]:
english_char_check('Instachat 😜')

False

The previous function didn't count in the apps with emojis or some characters. We will allow up to three characters so that we will be able to recognize English apps.

In [176]:
def is_english_v2(a_string):
    non_ascii = 0
    for character in a_string:
        if ord(character) > 127:
            non_ascii += 1
    
    if non_ascii > 3:
        return False
    else:
        return True    

In [177]:
is_english_v2('Docs To Go™ Free Office Suite')

True

In [178]:
is_english_v2('Instachat 😜')

True

In [179]:
is_english_v2('爱奇艺PPS -《欢乐颂2》电视剧热播')

False

In [180]:
android_english = []
ios_english = []

for row in android_clean:
    name = row[0]
    if is_english_v2(name) == True:
        android_english.append(row)

for row in ios:
    name = row[1]
    if is_english_v2(name) == True:
        ios_english.append(row)

print("No of Android Apps in English: " , len(android_english))
print("No of iOS Apps in English: " , len(ios_english))

No of Android Apps in English:  9614
No of iOS Apps in English:  6183


To find free apps

In [181]:
android_free_apps = []

for row in android_english:
    price = row[7]
    if price == '0':
        android_free_apps.append(row)

print("No of Free Android Apps in English: " , len(android_free_apps))

No of Free Android Apps in English:  8864


In [182]:
ios_free_apps = []
for row in ios_english:
    price = row[4]
    if price == '0.0':
        ios_free_apps.append(row)

print("No of Free iOS Apps in English: " , len(ios_free_apps))

No of Free iOS Apps in English:  3222


After cleaning data, we ended up with two lists, android_free_apps and ios_free_apps. We will use these list to performa analysis.

We want to find an app profile that fits both stores. Our validation strategy is to find out most common genres in both stores and build an app that fits both.

In [183]:
android_free_apps[0]

['Photo Editor & Candy Camera & Grid & ScrapBook',
 'ART_AND_DESIGN',
 '4.1',
 '159',
 '19M',
 '10,000+',
 'Free',
 '0',
 'Everyone',
 'Art & Design',
 'January 7, 2018',
 '1.0.0',
 '4.0.3 and up']

In [184]:
android_header[1]

'Category'

In [185]:
android_header[9]

'Genres'

In [186]:
ios_header[11]

'prime_genre'

In [187]:
def freq_table(dataset, index):
    frequency_dict = {}
    for row in dataset:
        col = row[index]
        if col in frequency_dict:
            frequency_dict[col] += 1
        else:
            frequency_dict[col] = 1
    return frequency_dict

In [188]:
def display_table(dataset, index):
    table = freq_table(dataset, index)
    table_display = []
    for key in table:
        key_val_as_tuple = (table[key], key)
        table_display.append(key_val_as_tuple)
    
    table_sorted = sorted(table_display, reverse = True)
    for entry in table_sorted:
        print(entry[1], ':', entry[0])

In [189]:
#Android - Category
display_table(android_free_apps, 1)

FAMILY : 1676
GAME : 862
TOOLS : 750
BUSINESS : 407
LIFESTYLE : 346
PRODUCTIVITY : 345
FINANCE : 328
MEDICAL : 313
SPORTS : 301
PERSONALIZATION : 294
COMMUNICATION : 287
HEALTH_AND_FITNESS : 273
PHOTOGRAPHY : 261
NEWS_AND_MAGAZINES : 248
SOCIAL : 236
TRAVEL_AND_LOCAL : 207
SHOPPING : 199
BOOKS_AND_REFERENCE : 190
DATING : 165
VIDEO_PLAYERS : 159
MAPS_AND_NAVIGATION : 124
FOOD_AND_DRINK : 110
EDUCATION : 103
ENTERTAINMENT : 85
LIBRARIES_AND_DEMO : 83
AUTO_AND_VEHICLES : 82
HOUSE_AND_HOME : 73
WEATHER : 71
EVENTS : 63
PARENTING : 58
ART_AND_DESIGN : 57
COMICS : 55
BEAUTY : 53


In [190]:
#Android - Genre
display_table(android_free_apps, 9)

Tools : 749
Entertainment : 538
Education : 474
Business : 407
Productivity : 345
Lifestyle : 345
Finance : 328
Medical : 313
Sports : 307
Personalization : 294
Communication : 287
Action : 275
Health & Fitness : 273
Photography : 261
News & Magazines : 248
Social : 236
Travel & Local : 206
Shopping : 199
Books & Reference : 190
Simulation : 181
Dating : 165
Arcade : 164
Video Players & Editors : 157
Casual : 156
Maps & Navigation : 124
Food & Drink : 110
Puzzle : 100
Racing : 88
Role Playing : 83
Libraries & Demo : 83
Auto & Vehicles : 82
Strategy : 81
House & Home : 73
Weather : 71
Events : 63
Adventure : 60
Comics : 54
Beauty : 53
Art & Design : 53
Parenting : 44
Card : 40
Casino : 38
Trivia : 37
Educational;Education : 35
Board : 34
Educational : 33
Education;Education : 30
Word : 23
Casual;Pretend Play : 21
Music : 18
Racing;Action & Adventure : 15
Puzzle;Brain Games : 15
Entertainment;Music & Video : 15
Casual;Brain Games : 12
Casual;Action & Adventure : 12
Arcade;Action & Advent

In [191]:
#iOS - Genre
display_table(ios_free_apps, 11)

Games : 1874
Entertainment : 254
Photo & Video : 160
Education : 118
Social Networking : 106
Shopping : 84
Utilities : 81
Sports : 69
Music : 66
Health & Fitness : 65
Productivity : 56
Lifestyle : 51
News : 43
Travel : 40
Finance : 36
Weather : 28
Food & Drink : 26
Reference : 18
Business : 17
Book : 14
Navigation : 6
Medical : 6
Catalogs : 4


After analysing prime_genre column of App Store dataset, I found that:
- Games is the most popular genre.
- Entertainment is the runner-up.
- Majority of English apps are in top 5 genres.
- Most of the apps are designed for fun purposes.
- Based on this frequency table, I would recommend to build apps around either games or entertainment. We can cross-check with downloads data (if there is any) to find out whether Games and Entertainment will have high number of users.

After analysing Category and Genres columns of Android dataset, I found that:

- Tools and Family apps are the most common genres.
- Faily distributed compared to App Store.
- Games and Entertainment apps not so dominant in Android.
- Most of the apps are designed for practical purposes.
- Based on this frequency table, I would recommend to build either Tools or Family related apps. We can cross-check with downloads data (if there is any) to find out whether these apps will have high number of users.

In [192]:
android_header[5]

'Installs'

In [193]:
ios_header[5]

'rating_count_tot'

To calculate the average number of user ratings per app genre on the App Store:

In [194]:
ios_prime_genre = freq_table(ios_free_apps, 11)

for genre in ios_prime_genre:
    total = 0
    len_genre = 0
    for row in ios_free_apps:
        genre_app = row[11]
        if genre_app == genre:
            num_ratings = float(row[5])
            total += num_ratings
            len_genre += 1
    avg_no_rating = total / len_genre
    print(genre, ':', avg_no_rating)

Photo & Video : 28441.54375
Lifestyle : 16485.764705882353
Weather : 52279.892857142855
Catalogs : 4004.0
Reference : 74942.11111111111
News : 21248.023255813954
Social Networking : 71548.34905660378
Medical : 612.0
Productivity : 21028.410714285714
Book : 39758.5
Music : 57326.530303030304
Sports : 23008.898550724636
Business : 7491.117647058823
Food & Drink : 33333.92307692308
Health & Fitness : 23298.015384615384
Finance : 31467.944444444445
Shopping : 26919.690476190477
Utilities : 18684.456790123455
Education : 7003.983050847458
Entertainment : 14029.830708661417
Travel : 28243.8
Navigation : 86090.33333333333
Games : 22788.6696905016


When we look at avg. number of reviews, navigation apps have the highest number of reviews. Let's deep-dive into these apps.

In [195]:
for app in ios_free_apps:
    if app[11] == 'Navigation':
        print(app[1], ':', app[5])

Waze - GPS Navigation, Maps & Real-time Traffic : 345046
Google Maps - Navigation & Transit : 154911
Geocaching® : 12811
CoPilot GPS – Car Navigation & Offline Maps : 3582
ImmobilienScout24: Real Estate Search in Germany : 187
Railway Route Search : 5


Waze and Google Maps dominate the app reviews. It is not a bad thing but it is not exactly what we need. Let's look at second highest category - Reference apps.

In [196]:
for app in ios_free_apps:
    if app[11] == 'Reference':
        print(app[1], ':', app[5])

Bible : 985920
Dictionary.com Dictionary & Thesaurus : 200047
Dictionary.com Dictionary & Thesaurus for iPad : 54175
Google Translate : 26786
Muslim Pro: Ramadan 2017 Prayer Times, Azan, Quran : 18418
New Furniture Mods - Pocket Wiki & Game Tools for Minecraft PC Edition : 17588
Merriam-Webster Dictionary : 16849
Night Sky : 12122
City Maps for Minecraft PE - The Best Maps for Minecraft Pocket Edition (MCPE) : 8535
LUCKY BLOCK MOD ™ for Minecraft PC Edition - The Best Pocket Wiki & Mods Installer Tools : 4693
GUNS MODS for Minecraft PC Edition - Mods Tools : 1497
Guides for Pokémon GO - Pokemon GO News and Cheats : 826
WWDC : 762
Horror Maps for Minecraft PE - Download The Scariest Maps for Minecraft Pocket Edition (MCPE) Free : 718
VPN Express : 14
Real Bike Traffic Rider Virtual Reality Glasses : 8
教えて!goo : 0
Jishokun-Japanese English Dictionary & Translator : 0


Again, we see here that Bible and dictionary apps have highest number of reviews. However, we get a trend here that reference apps / book related apps may be popular apps. So we can try to build one around that. Or we can follow the popular apps routes and build an app around games / social networking.

Time to find out Google Play Store data

In [197]:
android_unique_cat = freq_table(android_free_apps, 1)

for category in android_unique_cat:
    total = 0
    len_category = 0
    for row in android_free_apps:
        category_app = row[1]
        if category_app == category:
            installs_raw = row[5]
            installs_removed_plus = installs_raw.replace('+', '')
            installs_removed_com = installs_removed_plus.replace(',', '')
            installs_final = float(installs_removed_com)
            total += installs_final
            len_category += 1
    
    avg_installs = total / len_category
    print(category, ':', avg_installs)   

VIDEO_PLAYERS : 24727872.452830188
MEDICAL : 120550.61980830671
PERSONALIZATION : 5201482.6122448975
TOOLS : 10801391.298666667
EDUCATION : 1833495.145631068
FOOD_AND_DRINK : 1924897.7363636363
NEWS_AND_MAGAZINES : 9549178.467741935
MAPS_AND_NAVIGATION : 4056941.7741935486
COMICS : 817657.2727272727
GAME : 15588015.603248259
LIFESTYLE : 1437816.2687861272
PHOTOGRAPHY : 17840110.40229885
FINANCE : 1387692.475609756
PRODUCTIVITY : 16787331.344927534
SHOPPING : 7036877.311557789
DATING : 854028.8303030303
EVENTS : 253542.22222222222
ENTERTAINMENT : 11640705.88235294
HEALTH_AND_FITNESS : 4188821.9853479853
HOUSE_AND_HOME : 1331540.5616438356
BOOKS_AND_REFERENCE : 8767811.894736841
LIBRARIES_AND_DEMO : 638503.734939759
BUSINESS : 1712290.1474201474
AUTO_AND_VEHICLES : 647317.8170731707
FAMILY : 3695641.8198090694
SPORTS : 3638640.1428571427
WEATHER : 5074486.197183099
BEAUTY : 513151.88679245283
ART_AND_DESIGN : 1986335.0877192982
SOCIAL : 23253652.127118643
TRAVEL_AND_LOCAL : 13984077.7101

On average, communication apps have most number of installs. If we deep-dive into data:

In [198]:
for app in android_free_apps:
    if app[1] == 'COMMUNICATION' and (app[5] == '1,000,000,000+'
                                      or app[5] == '500,000,000+'
                                      or app[5] == '100,000,000+'):
        print(app[0], ':', app[5])

WhatsApp Messenger : 1,000,000,000+
imo beta free calls and text : 100,000,000+
Android Messages : 100,000,000+
Google Duo - High Quality Video Calls : 500,000,000+
Messenger – Text and Video Chat for Free : 1,000,000,000+
imo free video calls and chat : 500,000,000+
Skype - free IM & video calls : 1,000,000,000+
Who : 100,000,000+
GO SMS Pro - Messenger, Free Themes, Emoji : 100,000,000+
LINE: Free Calls & Messages : 500,000,000+
Google Chrome: Fast & Secure : 1,000,000,000+
Firefox Browser fast & private : 100,000,000+
UC Browser - Fast Download Private & Secure : 500,000,000+
Gmail : 1,000,000,000+
Hangouts : 1,000,000,000+
Messenger Lite: Free Calls & Messages : 100,000,000+
Kik : 100,000,000+
KakaoTalk: Free Calls & Text : 100,000,000+
Opera Mini - fast web browser : 100,000,000+
Opera Browser: Fast and Secure : 100,000,000+
Telegram : 100,000,000+
Truecaller: Caller ID, SMS spam blocking & Dialer : 100,000,000+
UC Browser Mini -Tiny Fast Private & Secure : 100,000,000+
Viber Mess

The number is heavily screwed by Facebook family apps.

We see the same pattern for the video players category, which is the runner-up with 24,727,872 installs. The market is dominated by apps like Youtube, Google Play Movies & TV, or MX Player. The pattern is repeated for social apps (where we have giants like Facebook, Instagram, Google+, etc.), photography apps (Google Photos and other popular photo editors), or productivity apps (Microsoft Word, Dropbox, Google Calendar, Evernote, etc.).
Again, the main concern is that these app genres might seem more popular than they really are. Moreover, these niches seem to be dominated by a few giants who are hard to compete against.
The game genre seems pretty popular, but previously we found out this part of the market seems a bit saturated, so we'd like to come up with a different app recommendation if possible.
The books and reference genre looks fairly popular as well, with an average number of installs of 8,767,811. It's interesting to explore this in more depth, since we found this genre has some potential to work well on the App Store, and our aim is to recommend an app genre that shows potential for being profitable on both the App Store and Google Play.

In [199]:
for app in android_free_apps:
    if app[1] == 'BOOKS_AND_REFERENCE':
        print(app[0], ':', app[5])

E-Book Read - Read Book for free : 50,000+
Download free book with green book : 100,000+
Wikipedia : 10,000,000+
Cool Reader : 10,000,000+
Free Panda Radio Music : 100,000+
Book store : 1,000,000+
FBReader: Favorite Book Reader : 10,000,000+
English Grammar Complete Handbook : 500,000+
Free Books - Spirit Fanfiction and Stories : 1,000,000+
Google Play Books : 1,000,000,000+
AlReader -any text book reader : 5,000,000+
Offline English Dictionary : 100,000+
Offline: English to Tagalog Dictionary : 500,000+
FamilySearch Tree : 1,000,000+
Cloud of Books : 1,000,000+
Recipes of Prophetic Medicine for free : 500,000+
ReadEra – free ebook reader : 1,000,000+
Anonymous caller detection : 10,000+
Ebook Reader : 5,000,000+
Litnet - E-books : 100,000+
Read books online : 5,000,000+
English to Urdu Dictionary : 500,000+
eBoox: book reader fb2 epub zip : 1,000,000+
English Persian Dictionary : 500,000+
Flybook : 500,000+
All Maths Formulas : 1,000,000+
Ancestry : 5,000,000+
HTC Help : 10,000,000+
E

This niche seems to be dominated by software for processing and reading ebooks, as well as various collections of libraries and dictionaries, so it's probably not a good idea to build similar apps since there'll be some significant competition.
We also notice there are quite a few apps built around the book Quran, which suggests that building an app around a popular book can be profitable. It seems that taking a popular book (perhaps a more recent book) and turning it into an app could be profitable for both the Google Play and the App Store markets.
However, it looks like the market is already full of libraries, so we need to add some special features besides the raw version of the book. This might include daily quotes from the book, an audio version of the book, quizzes on the book, a forum where people can discuss the book, etc.

## Conclusions

In this project, we analyzed data about the App Store and Google Play mobile apps with the goal of recommending an app profile that can be profitable for both markets.
We concluded that taking a popular book (perhaps a more recent book) and turning it into an app could be profitable for both the Google Play and the App Store markets. The markets are already full of libraries, so we need to add some special features besides the raw version of the book. This might include daily quotes from the book, an audio version of the book, quizzes on the book, a forum where people can discuss the book, etc.